### PR TITLE
feat: DB Branching

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

File: docs/sdkstools/typescript/database/branching.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+```ts
+const tigrisClient = new Tigris();
+```
+
+</TabItem>
+</Tabs>
+
+This way all the schema changes and database queries would be against the `development` branch. When
+a database is created a `main` branch always exist which essentially is the primary database branch
```
Comment (himank) [inline docs/sdkstools/typescript/database/branching.mdx:49]: we can remove this branch and can just refer everywhere that the `main branch is essentially the primary database`.

File: docs/concepts/database/branching.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+With Tigris you can branch your database schema similar to how you branch your git repository. You
+can have multiple isolated copies of your database schema to develop and test your features with.
+Additionally, you can have your CI/CD pipelines use isolated data branches for execution without any
+overhead of setting up and maintaining a development database.
+
+## Branch
+
+Similar to a git repo, your Tigris database always defaults to `main` as primary source of data and schema.
+`main` branch is already created in your Tigris Project's database and it cannot be deleted. All new
```
Comment (himank) [inline docs/concepts/database/branching.md:11]: Something like this `main branch is referred to the primary database of the project`

Comment (adilansari) [reply docs/concepts/database/branching.md]: Updated as part of Phil's suggested commit

**Comment by adilansari** [comment]
This looks much better.

**Comment by ovaistariq** [comment]
> This looks much better. 

I just tweaked a few things, it's mainly all you

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**